jpa/hibernatesearch/src/main/java/org/jipijapa/hibernate/search/WildFlyHibernateOrmSearchMappingConfigurer.java

```diff
@@ -0,0 +1,30 @@
+package org.jipijapa.hibernate.search;
+
+import org.hibernate.search.mapper.orm.mapping.HibernateOrmMappingConfigurationContext;
+import org.hibernate.search.mapper.orm.mapping.HibernateOrmSearchMappingConfigurer;
+import org.jboss.jandex.Index;
+
+import java.util.Collection;
+
+final class WildFlyHibernateOrmSearchMappingConfigurer implements HibernateOrmSearchMappingConfigurer {
+    private final Collection<Index> indexes;
+
+    public WildFlyHibernateOrmSearchMappingConfigurer(Collection<Index> indexes) {
+        this.indexes = indexes;
+    }
+
+    @Override
+    public void configure(HibernateOrmMappingConfigurationContext context) {
+        // Hibernate Search can't deal with WildFly's JAR URLs using vfs://,
+        // so it fails to load Jandex indexes from the JARs.
+        // Regardless, WildFly already has Jandex indexes in memory,
+        // so we'll configure Hibernate Search to just use those.
```

Comment:
WildFly only has access to the Jandex indexes at application deployment time.  After the application deploys, Hibernate Search should not have any references to the Jandex indexes since they can be quite large.